tests/helpers/test_translation.py

```diff
@@ -267,3 +267,11 @@ async def test_caching(hass):
 
         await translation.async_get_translations(hass, "en", "state")
         assert len(mock_merge.mock_calls) == 2
+
+
+async def test_custom_component_translations(hass):
+    """Test getting translation from custom components."""
+    hass.config.components.add("test_standalone")
+    hass.config.components.add("test_embedded")
+    hass.config.components.add("test_package")
+    assert await translation.async_get_translations(hass, "en", "state") == {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5456697)